docs/topics/class-based-views/index.txt

```diff
@@ -76,7 +76,8 @@ method instead, which provides a function-like entry to class-based views::
 
     # urls.py
     from django.urls import path
-    from some_app.views import AboutView
+
+    from .views import AboutView
```

Comment:
Leave alone. This looks like a root URLConf.

Reply:
ok